content/docs/rendering-elements.md

```diff
@@ -18,58 +18,58 @@ const element = <h1>Hola, mundo</h1>;
 
 A diferencia de los elementos del DOM de los navegadores, los elementos de React son objetos planos, y su creación es de bajo costo. React DOM se encarga de actualizar el DOM para igualar los elementos de React.
 
->**Note:**
+>**Nota:**
 >
->One might confuse elements with a more widely known concept of "components". We will introduce components in the [next section](/docs/components-and-props.html). Elements are what components are "made of", and we encourage you to read this section before jumping ahead.
+>Uno podría confundir los elementos con el muy conocido concepto de "componentes". En la siguiente sección [siguiente sección](/docs/components-and-props.html) hablaremos de componentes. Los elementos son los que "constituyen" los componentes, y recomendamos leer esta sección antes de continuar.
 
-## Rendering an Element into the DOM
+## Renderizando un Elemento en el DOM
 
-Let's say there is a `<div>` somewhere in your HTML file:
+Digamos que hay un `<div>` en alguna parte de tu archivo HTML:
 
 ```html
 <div id="root"></div>
 ```
 
-We call this a "root" DOM node because everything inside it will be managed by React DOM.
+Nosotros lo llamamos un nodo "raíz" porque todo lo que esté dentro de él será manejado por React DOM.
 
-Applications built with just React usually have a single root DOM node. If you are integrating React into an existing app, you may have as many isolated root DOM nodes as you like.
+Las aplicaciones construidas con solo React usualmente tienen un único nodo raíz en el DOM. Dado el caso que estés integrando React en una aplicación existente, podrías tener cuantos nodos raíz en el DOM como quieras.
 
-To render a React element into a root DOM node, pass both to `ReactDOM.render()`:
+Para renderizar un elemento de React en el nodo raíz del DOM, ambos se pasan a `ReactDOM.render()`:
```

Comment:
I think "_pass both to_" is written in imperative, so it would translate to `pasa ambos a`.

Also worth a note, it says "a root DOM node" rather than "the root DOM node", so perhaps it would be better to use `en un nodo raíz del DOM`. Changing `the` by `un`.

What do you think?

Reply:
Sounds good! 👍